corehq/apps/hqwebapp/tasks.py

```diff
@@ -242,11 +242,13 @@ def clean_expired_transient_emails():
         )
 
 
+# ToDo: remove/modify this if the related metric is removed/retained
```

Comment:
Got a confirmation from SaaS (Danny) that this was okay to remove. So this was removed later.